ext/js/data/options-util.js

```diff
@@ -114,6 +114,7 @@ class OptionsUtil {
 
         if (typeof options !== 'undefined') {
             options = await this.update(options);
+            await save(options);
```

Comment:
Lint failure... should this be `await this.save(options)`?

Reply:
right, sorry. forgot to run tests.